doc/classes/Image.xml

```diff
@@ -64,13 +64,14 @@
 			<argument index="3" name="dst" type="Vector2">
 			</argument>
 			<description>
-				Blits a "src_rect" [Rect2] from "src" [code]Image[/code] to this [code]Image[/code] using a "mask" [code]Image[/code] on coordinates "dest". Alpha channel is required for "mask", will copy src pixel onto dest if the corresponding mask pixel's alpha value is not 0. "src" [code]Image[/code] and "mask" [code]Image[/code] *must* have the same size (width and height) but they can have different formats
+				Blits [code]src_rect[/code] area from [code]src[/code] image to this image on the coordinates specified by [code]dest[/code]. Uses a [code]mask[/code] image for deciding if a pixel is blit or not. Alpha channel is required for [code]mask[/code], will copy [code]src[/code] pixel onto [code]dest[/code] if the corresponding mask pixel's alpha value is not 0. [code]src[/code] image and [code]mask[/code] image *must* have the same size (width and height) but they can have different formats.
```

Comment:
- on the coordinates -> at the coordinate
- must -> [b]must[/b]

> Uses a [code]mask[/code] image for deciding if a pixel is blit or not.

I think you can remove this sentence, as the next one explains the mask anyway.

> Alpha channel is required for [code]mask[/code], will copy [code]src[/code] pixel onto [code]dest[/code] if the corresponding mask pixel's alpha value is not 0.

This sentence is run-on.
"[code]src[/code] pixel is copied onto [code]dst[/code] if the corresponding [code]mask[/code] pixel's alpha value is not 0.